src/modules/dashboard/components/ActionsPage/CountDownTimer/CountDownTimer.tsx

```diff
@@ -4,6 +4,7 @@ import React, {
   useCallback,
   useRef,
   MutableRefObject,
+  useMemo,
 } from 'react';
 import { FormattedMessage, defineMessage } from 'react-intl';
 import { useDispatch } from 'redux-react-hook';
```

Comment:
There's a issue in this component, after the timeout runs out, it tries to trigger a motion values update (by calling `dispatch`)

But the `motionId` is just a `string`, at it's expecting a `bigNumber`

![Screenshot from 2021-06-16 16-25-07](https://user-images.githubusercontent.com/1193222/122227671-c559bc80-cebf-11eb-9b43-c5344d0e9173.png)

Please convert it to a `bigNumber` using `bigNumberify` before passing it to `dispatch`